src/datetime-score-api/tickers.txt

```diff
@@ -0,0 +1 @@
"Micron", "MU"], "Microsemi": ["Microsemi", "MSCC"], "Microsoft": ["Microsoft", "MSFT"], "Middleby": ["Middleby", "MIDD"], "MKS": ["MKS", "MKSI"], "Momo": ["Momo", "MOMO"], "Mondelez": ["Mondelez", "MDLZ"], "Monster": ["Monster", "MNST"], "Mylan": ["Mylan", "MYL"], "Nasdaq": ["Nasdaq", "NDAQ"], "National": ["National", "NATI"], "Nektar": ["Nektar", "NKTR"], "NetApp": ["NetApp", "NTAP"], "NetEase": ["NetEase", "NTES"], "Netflix": ["Netflix", "NFLX"], "Neurocrine": ["Neurocrine", "NBIX"], "News": ["News", "NWSA"], "Nordson": ["Nordson", "NDSN"], "Northern": ["Northern", "NTRS"], "Norwegian": ["Norwegian", "NCLH"], "Nutanix": ["Nutanix", "NTNX"], "NVIDIA": ["NVIDIA", "NVDA"], "NXP": ["NXP", "NXPI"], "Old": ["Old", "ODFL"], "ON": ["ON", "ON"], "Open": ["Open", "OTEX"], "O'Reilly": ["O'Reilly", "ORLY"], "PACCAR": ["PACCAR", "PCAR"], "PacWest": ["PacWest", "PACW"], "Paychex": ["Paychex", "PAYX"], "PayPal": ["PayPal", "PYPL"], "People's": ["People's", "PBCT"], "Pepsico": ["Pepsico", "PEP"], "Pilgrim's": ["Pilgrim's", "PPC"], "Pinnacle": ["Pinnacle", "PNFP"], "Pool": ["Pool", "POOL"], "PowerShares": ["PowerShares", "QQQ"], "PRA": ["PRA", "PRAH"], "Principal": ["Principal", "PFG"], "Proofpoint": ["Proofpoint", "PFPT"], "PTC": ["PTC", "PTC"], "Qiagen": ["Qiagen", "QGEN"], "Qorvo": ["Qorvo", "QRVO"], "QUALCOMM": ["QUALCOMM", "QCOM"], "Regeneron": ["Regeneron", "REGN"], "Ross": ["Ross", "ROST"], "Royal": ["Royal", "RGLD"], "Ryanair": ["Ryanair", "RYAAY"], "Sabre": ["Sabre", "SABR"], "Sage": ["Sage", "SAGE"], "SBA": ["SBA", "SBAC"], "Seagate": ["Seagate", "STX"], "Seattle": ["Seattle", "SGEN"], "SEI": ["SEI", "SEIC"], "Shire": ["Shire", "SHPG"], "Signature": ["Signature", "SBNY"], "Sina": ["Sina", "SINA"], "Sirius": ["Sirius", "SIRI"], "Skyworks": ["Skyworks", "SWKS"], "Splunk": ["Splunk", "SPLK"], "SS&C": ["SS&C", "SSNC"], "Starbucks": ["Starbucks", "SBUX"], "Steel": ["Steel", "STLD"], "Stericycle": ["Stericycle", "SRCL"], "SVB": ["SVB", "SIVB"], "Symantec": ["Symantec", "SYMC"], "Synopsys": ["Synopsys", "SNPS"], "SPDR": ["SPDR", "SPY"], "T.": ["T.", "TROW"], "Take-Two": ["Take-Two", "TTWO"], "TD": ["TD", "AMTD"], "Tesla": ["Tesla", "TSLA"], "Texas": ["Texas", "TXN"], "T-Mobile": ["T-Mobile", "TMUS"], "Tractor": ["Tractor", "TSCO"], "Trimble": ["Trimble", "TRMB"], "TripAdvisor": ["TripAdvisor", "TRIP"], "Twenty-First": ["Twenty-First", "FOX"], "Ubiquiti": ["Ubiquiti", "UBNT"], "Ulta": ["Ulta", "ULTA"], "Ultimate": ["Ultimate", "ULTI"], "VeriSign": ["VeriSign", "VRSN"], "Verisk": ["Verisk", "VRSK"], "Vertex": ["Vertex", "VRTX"], "Viacom": ["Viacom", "VIA"], "Virtu": ["Virtu", "VIRT"], "Vodafone": ["Vodafone", "VOD"], "Walgreens": ["Walgreens", "WBA"], "Weibo": ["Weibo", "WB"], "Western": ["Western", "WDC"], "Willis": ["Willis", "WLTW"], "Workday": ["Workday", "WDAY"], "Wynn": ["Wynn", "WYNN"], "Xcel": ["Xcel", "XEL"], "Xilinx": ["Xilinx", "XLNX"], "Yandex": ["Yandex", "YNDX"], "Zebra": ["Zebra", "ZBRA"], "Zillow": ["Zillow", "ZG"], "Zions": ["Zions", "ZION"]}
```

Comment:
It's probably not necessary to duplicate the name of every company in this list. To get the stock symbol we have to know the company name, and if we already know the name there isn't a need to have it in the returned array.

Reply:
Fixed.